data_targets/subnationals/CDP Cities targets/Harmonized/EmissionsAgg.csv

```diff
@@ -0,0 +1,844 @@
+"actor_id","emissions_id","year","total_emissions","datasource_id"
+"CO BOG","CDPCitiesTargets:2021:base:CO BOG",2017,11424724,"CDPCitiesTargets:2021"
+"CO BOG","CDPCitiesTargets:2021:BAU:CO BOG",2024,13438701,"CDPCitiesTargets:2021"
+"GB MNC","CDPCitiesTargets:2021:base:GB MNC",1990,21200000,"CDPCitiesTargets:2021"
+"GB MNC","CDPCitiesTargets:2021:target:GB MNC",2038,572400,"CDPCitiesTargets:2021"
+"ES MJV","CDPCitiesTargets:2021:base:ES MJV",2007,2141277,"CDPCitiesTargets:2021"
+"ES MJV","CDPCitiesTargets:2021:target:ES MJV",2020,1713021.6,"CDPCitiesTargets:2021"
```

Comment:
please check column type. total_emissions needs to be an integer

Reply:
@lgloege New commit fixes the following issues: 

- some of the emissions_total were floats, they need to be integers.
- Some columns don’t match the schema. For example, “City.location” in Territory
- add a Tag table where you define your tag_ids. I don’t think we have a tag yet for “self_reported”, I think that’s a new one
- please clean up the Target table. For example, if it doesn’t have a target_year , base_year and  target_value, we don’t need it